src/test/java/CalendarServletTest.java

```diff
@@ -90,29 +67,19 @@
           new Event().setSummary(EVENT_SUMMARY_ONE),
           new Event().setSummary(EVENT_SUMMARY_TWO));
 
+  @Override
   @Before
-  public void setUp() throws IOException, GeneralSecurityException {
-    authenticationVerifier = Mockito.mock(AuthenticationVerifier.class);
+  public void setUp() throws Exception {
+    super.setUp();
     calendarClientFactory = Mockito.mock(CalendarClientFactory.class);
     calendarClient = Mockito.mock(CalendarClient.class);
     servlet = new CalendarServlet(authenticationVerifier, calendarClientFactory);
 
     Mockito.when(calendarClientFactory.getCalendarClient(Mockito.any())).thenReturn(calendarClient);
-    // Authentication will always pass
-    Mockito.when(authenticationVerifier.verifyUserToken(Mockito.anyString())).thenReturn(true);
-
-    request = Mockito.mock(HttpServletRequest.class);
-    response = Mockito.mock(HttpServletResponse.class);
-    Mockito.when(request.getCookies()).thenReturn(validCookies);
-
-    // Writer used in get/post requests to capture HTTP response values
-    stringWriter = new StringWriter();
-    printWriter = new PrintWriter(stringWriter);
-    Mockito.when(response.getWriter()).thenReturn(printWriter);
   }
 
   @Test
-  public void noCalendarEvent() throws IOException, ServletException {
+  public void noCalendarEvent() throws Exception {
```

Comment:
I received this review comment from either Gui or Anton on one of my PRs - throwing IOException and ServletException like before is better because it's more descriptive than just Exception.

Reply:
@gbuenoandrade informed me in a [PR](https://github.com/googleinterns/step7-2020/pull/86#discussion_r458278603) that you can just throw Exception in tests. 

This part of the refactor is far from needed, so I can undo this if anyone is adamantly against it, but if it's permissible for tests then I don't see a reason not to.

Reply:
That's fair! I just noticed that the files where you changed these are all tests.